docs/userstories/ClusterNetworkPolicy/Funnel_Cluster_Traffic.md

```diff
@@ -0,0 +1,22 @@
+### Summary
```

Comment:
In general I think this story may be impossible, and or impossible to implement from a CNI's perspective.  

For example what would be done in the case a developer creates an external facing service (Nodeport, externalIP, ect.) backed by host networked pods? 

Here traffic could gress the cluster freely, and since CNI's can't _see_ host networked pods network policy would have no effect.

Reply:
The solution for my concern is actually somewhat simple, we should just should clarify something like this for every ingress/egress related story : 

```
An administrator should be able to allow all (non-privileged, non-hostNetwork) 
pods in a cluster to send/receive external traffic  only via specified gateways.
```